src/media/css/components/addon-listing-dashboard.styl

```diff
@@ -0,0 +1,77 @@
+@import '../lib'
+
+
+.addon-dashboard .page--main
+.addon-dashboard h1
+  max-width 800px
```

Comment:
Nope, I'm eyeballing what looks not-completely-terrible. There's not enough content here to justify a full 1024.